src/vs/platform/terminal/node/terminalProcess.ts

```diff
@@ -63,9 +66,10 @@ export class TerminalProcess extends Disposable implements ITerminalChildProcess
 
 	private _isPtyPaused: boolean = false;
 	private _unacknowledgedCharCount: number = 0;
-
 	public get exitMessage(): string | undefined { return this._exitMessage; }
+
 	public get currentTitle(): string { return this._currentTitle; }
+	public get lastTitle(): string { return this._windowsShellHelper?.lastShellTitle || this._currentTitle; }
```

Comment:
I couldn't get it to work using only currentTitle in a way that's similar to the shellType.